pkgs/os-specific/linux/lkvm/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv, fetchgit, libbfd, zlib, libaio, glibc }:
+
+stdenv.mkDerivation {
+  name = "lkvm";
+  version = "2020-05-19";
```

The name should denote this is not an official version:

```suggestion
  version = "unstable-2020-05-19";
```